Mutagen.Bethesda.Analyzers.Cli/Args/RunAnalyzersCommand.cs

```diff
@@ -10,6 +11,9 @@ public class RunAnalyzersCommand : IMinimumSeverityConfiguration
     [Option('g', "GameRelease", Required = true, HelpText = "Game Release to target")]
     public GameRelease GameRelease { get; set; }
 
+    [Option('l', "Language", Required = false, HelpText = "Language to target, defaults to English")]
+    public Language Language { get; set; } = Language.English;
```

Comment:
Can you walk me through how this is being used?

Reply:
It's injected into the DI container and then can be used to be resolved in each analyzer where needed.

Reply:
Hmm, I'd want to maybe have a discussion on language overall for the analyzers project.   Does everyone pick one language and the analyzers always run against that single?  Or do analyzers handle multiple languages by default, and we filter them out somehow via config?

Not quite sure offhand.  But I think this solution is going the route where analyzers wouldnt be able to look at all languages simultaneously?   Seems a bit limiting for users that want to analyze overall correctness across the board

Reply:
I can try checking all languages by default

Reply:
Seems better. It also needs all the game languages now, take a look :)